frontend-web/webclient/app/Admin/Overview.tsx

```diff
@@ -16,6 +16,7 @@ const linkInfo: LinkInfo[] = [
     {to: "/admin/appk8", text: "Maintenance (UCloud)", icon: "calendar", color: "white", color2: "black"},
     {to: "/admin/providers", text: "Providers", icon: "cloudTryingItsBest", color: "white", color2: "white"},
     {to: "/admin/scripts", text: "Scripts", icon: "play", color: "white", color2: "white"},
+    {to: "/admin/devData", text: "Development Test Data", icon: "activity", color: "white", color2: "white"},
 ];
 
```

Comment:
```suggestion
];

if (DEVELOPMENT_ENVIROMENT) {
    linkInfo.push({to: "/admin/devData", text: "Development Test Data", icon: "activity", color: "white", color2: "white"}),
}

```

Reply:
I believe `devData` only makes sense on the `dev` environment, so I wouldn't allow it elsewhere, if that is the case.

Note: The predicate is not correct in this suggestion.